packages/core/jest.config.js

```diff
@@ -12,6 +12,6 @@ module.exports = {
     '^@/(.*)$': '<rootDir>/src/$1',
     '^jose/(.*)$': '<rootDir>/node_modules/jose/dist/node/cjs/$1',
   },
-  coveragePathIgnorePatterns: ['/node_modules/', '/build/'],
+  coveragePathIgnorePatterns: ['/node_modules/', '/build/', '/src/__mocks__/'],
```

Comment:
nit: can we move `__mocks__` to the root dir? @logto-io/eng

Reply:
It's a best practice to put `__mocks__` folder immediately adjacent to the target module. E.g.

<img width="971" alt="image" src="https://user-images.githubusercontent.com/12833674/162368649-3e35a685-9332-46be-a40a-8f2fabf825d9.png">

So I don't know, maybe we should even break it into several more `__mocks__` folders?

Reply:
@simeng-li any idea

Reply:
Conclusion: Keep it for now, and build excludes `src/__mocks__/`.

See Slack [thread](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1649389597380479)

Reply:
Updated. PTAL @wangsijie @demonzoo